testbeam_analysis/tools/plot_utils.py

```diff
@@ -193,6 +193,7 @@ def plot_cluster_size(input_cluster_file, dut_name=None, output_pdf_file=None, c
         ax.set_yscale('linear')
         ax.set_ylim(ymax=np.amax(hight))
         ax.set_xlim(0.5, min(10, max_cluster_size) + 0.5)
+        output_pdf.savefig(fig)
```

Comment:
@DavidLP As it looks to me, all the lines added in this commit, due to merging, are incorrect.